chalicelib/app.py

```diff
@@ -180,19 +196,37 @@ def auth0_callback():
     return app_utils_obj.auth0_callback(request, DEFAULT_ENV)
 
 
-@app.route('/', methods=['GET'])
+if CHALICE_LOCAL:
```

Comment:
Maybe we should have a decorator that does this so that the definition doesn't have to be indented. Either by subclassing Chalice with something so that app could be `app = C4Chalice(...)` and we do `@app.chalice_local_route` because we have a `chalice_local_route` method defined. or something like that.
Just a thought. I guess since it only happens one place, it's not so bad, although we could also arrange for a decorator that didn't have to use those `ROUTE_PREFIX` prefixes. Just thinking aloud. Not high priority maybe.